### PR TITLE
Operators:  `.contains`, `.startsWith`, `.endsWith`

### PR DESCRIPTION
**Pull request by paulrolfe**
- Added the query operators `.contains`, `.startsWith`, `.endsWith` (which only work with Strings)
- Main goal was to get closer parity to Vapor operators.
- ~~Needs tests~~

### CODE REVIEW

File: Sources/MongoKitten/QueryBuilder.swift [hunk 1]
```diff
@@ -281,6 +287,15 @@ public indirect enum AQT {
     
     /// Whether nothing needs to be matched. Is always true and just a placeholder
     case nothing
+    
+    /// Whether the String value within the `key` contains this `Value` as a String.
+    case contains(key: String, val: ValueProtocol)
```
Comment (Joannis) [inline Sources/MongoKitten/QueryBuilder.swift]: Why would you choose `ValueProtocol` above `String` like `contains(key: String, val: String)`?

In the end you're only validating String Values.

Comment (paulrolfe) [inline Sources/MongoKitten/QueryBuilder.swift]: Good point. I was just sticking with the established pattern that took values. I'll switch it to `String`.

**Comment by Joannis** [comment]
I don't have any other comments on this PR anymore. @Obbut ?

**Comment by paulrolfe** [comment]
Should the Travis CI build be working?

**Comment by Joannis** [comment]
It shouldn't because we're unable to get MongoDB server set up on Linux using travis

**Comment by paulrolfe** [comment]
Cool are we ready to merge this?

**Comment by Obbut** [comment]
Done! Sorry for the late response. I'll create a patch release right now.

**Pull request merged**